images/fedora/f39/Containerfile

```diff
@@ -27,6 +27,8 @@ COPY extra-packages /
 RUN dnf -y install $(<extra-packages)
 RUN rm /extra-packages
 
+RUN ln -s /usr/bin/flatpak-xdg-open /usr/bin/xdg-open
```

Comment:
Nitpick: `ln --symbolic ...`, instead of `ln -s` because the long options are easier to `grep(1)`.

Reply:
I have updated the `ln` call to use the long option instead.